lib/laa_crime_schemas/structs/partner.rb

```diff
@@ -6,7 +6,9 @@ class Partner < Person
       attribute? :date_of_birth, Types::JSON::Date.optional
       attribute? :benefit_type, Types::BenefitType.optional
       attribute? :last_jsa_appointment_date, Types::JSON::Date.optional
+      attribute? :has_nino, Types::String.optional
```

Comment:
We were submitting the has_nino attribute but this was actually missing in the schema for the partner. As it's not required it wasn't blowing up